docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,42 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector.
```

Comment:
@dain is the new catalog available for new queries immediately after this query finishes, or is it loaded asynchronously? Either way, is it worth mentioning here?

Reply:
It is immediately available when the command returns.  Actually constructing the connector is pretty fast.  When you run queries for the first time you can get some slowness as connections are established, and some connectors cache information and the caches would be empty